main-brunch.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
```

Comment:
You can delete other html files and rename this to index.html. It will be your main file, so here you will be making your project

Reply:
done